helpers/password_helper.go

```diff
@@ -0,0 +1,12 @@
+package helpers
+
+import "golang.org/x/crypto/bcrypt"
+
+// EncryptPassword encrypts plain password with bcrypt adaptive hashing algorithm.
+func EncryptPassword(plainPassword string) (encryptedPassword string, err error) {
```

Comment:
`bcrypt` actually does hashing, not encrypting. So how about rename it to `HashPassword`?

Reply:
Changed ✅